helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -604,11 +607,8 @@ private void subscribeForChanges(NotificationContext.Type callbackType, String p
     }
 
     if (_eventTypes.contains(EventType.NodeChildrenChanged)) {
-      logger.info("Subscribing child change listener to path: {}", path);
```

Comment:
I personally think these 2 info logs are still desired. Especially we are changing the subscription mechanism.